src/main/java/frc/robot/commands/vision/TestVisionDistance.java

```diff
@@ -60,15 +67,22 @@ public void execute(){
       // reset the timer
       m_endTimer.reset();
       // If kPrintDelay seconds have passed, print the data
-      // TODO: Consider using shuffleboard instead of printing
+      double driveDistance = m_drive.getPose().getTranslation().getDistance(m_driveStartTranslation);
+      double visionDistance = m_currentPose.getTranslation().getDistance(m_visionStartTranslation);
+      m_outputString=String.format("\nEncoder distance: %.4f\nVision distance: %.4f\n"+
+        "Difference: %.4f\nPercent difference: %.4f%%",
+        driveDistance, visionDistance,
+        visionDistance - driveDistance, (visionDistance - driveDistance) / driveDistance * 100
+      );
       if (m_printTimer.advanceIfElapsed(kPrintDelay)) {
-        double driveDistance = m_drive.getPose().getTranslation().getDistance(m_driveStartTranslation);
-        double visionDistance = m_currentPose.getTranslation().getDistance(m_visionStartTranslation);
-        System.out.printf("\nEncoder distance: %.4f\nVision distance: %.4f\n",
-          driveDistance, visionDistance);
-        System.out.printf("Difference: %.4f\nPercent difference: %.4f%%\n",
-          visionDistance - driveDistance, (visionDistance - driveDistance) / driveDistance * 100);
-      } 
+        System.out.println(m_outputString);
+      }
+      if(Constants.kLogging){
+        LogManager.addDoubleArray("Vision/DistanceTest", new double[]{
+          driveDistance, visionDistance, 
+          visionDistance - driveDistance, (visionDistance - driveDistance) / driveDistance * 100
+        });
```

Comment:
It may be benificial to have seprate log entries rather than useing a double so it is clear wich messurement is which

Reply:
If I did that, would it keep all 4 numbers together or put them on the same row in a table? They aren't useful unless they can be compared to each other.

Reply:
you can indicudualy drag them in to compare them in advantage scope. But it is really dificult to compare things that you aren't sure what they are.